magda-web-client/src/Components/Dataset/Add/Pages/DetailsAndContents/index.tsx

```diff
@@ -75,8 +75,9 @@ export default function DatasetAddAccessAndUsePage(props: Props) {
                             value={dataset.title}
                             onChange={editDataset("title")}
                             editor={textEditorEx({
-                                placeholder:
-                                    !dataset.title && "Enter dataset title",
+                                placeholder: dataset.title
```

Comment:
@sajidanower23 
Had another test and it seems now placeholder won't be shown anymore.
Seems the logic here should be reversed

Reply:
Good catch @t83714 . Switched the logic around. Should work now.